### DISCUSSION TITLE
May I ask if the rand3 kernel display in AMD's CPU can use rocm?

### DISCUSSION THREAD

**Discussion by jungyin** [ROCm/ROCm#2999] [Q&A] [answered]
Recently, I have been selecting my new computer and would like to purchase an 8845 laptop. However, I am not sure if this computer's graphics card can be used with ROCM. Can someone with experience guide me?

**Comment by taylding-amd** [comment, marked as answer]
Hi @jungyin, for graphic card compatibility, please refer to the [compatibility metric](https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html#mi300-624) in AMD's official document. Meanwhile, Mobile GPUs (e.g., those in laptops) or integrated GPUs are not always officially supported by ROCm, even if their desktop counterparts are. Performance and compatibility might vary significantly.
